src/overmap.cpp

```diff
@@ -3174,9 +3174,6 @@ void overmap::build_anthill( const tripoint &p, int s )
             }
         }
     }
-    if( queenpoints.empty() ) {
-        debugmsg( "No queenpoints when building anthill" );
-    }
     const tripoint target = random_entry( queenpoints );
```

Comment:
`random_entry` will return a default constructed value, in that case (0,0,0) - so the queen chamber will always be at that point within each overmap - **irregardless** of the location of the anthill.

It would probably be best to skip setting the terrain "ants_queen" in that case.

Reply:
Done.